integration/src/test/scala/XmlSchemaTest.scala

```diff
@@ -15,6 +15,7 @@ object XmlSchemaTest extends TestBase {
   lazy val generated = module.process(inFile, config)
   "XMLSchema.scala file must compile so that Schema can be used" in {
     (List("import scalaxb._",
+      "import scalaxb.protocol._",
```

Comment:
If you had to do change this here, it implies that all existing scalaxb users would have to do the same, and it's actually a nonsensical value (`scalaxb.protocol`). Shouldn't the protocol package fall back to the package used for case classes?